### DISCUSSION TITLE
Does ImageMagick support the JPEG XR image?

### DISCUSSION THREAD

**Discussion by Charltsing** [dlemstra/Magick.NET#1750] [Help]
### Magick.NET version

13.10.0

### Environment (Operating system, version and so on)

Windows10, net framework 4.7.2

### Description

Why is the JPEG XR image format not supported
[fixture.jxr.zip](https://github.com/user-attachments/files/17687848/fixture.jxr.zip)

### Steps to Reproduce

...

**Comment by dlemstra** [comment]
There is no support for the JPEG XR format in the ImageMagick library and I don't think we will be going to add support for this anytime soon. You will need to find another tool to convert it to a format that ImageMagick understands.

**Comment by dlemstra** [comment]
Next time please create a discussion instead when you have a question.
